backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/picking/api/Packageable.java

```diff
@@ -159,7 +159,6 @@ public Quantity getQtyPickedOrDelivered()
 	public Quantity getQtyToPick()
 	{
 		return qtyToDeliver
-				.subtract(qtyPickedNotDelivered)
 				// IMPORTANT: don't subtract the Qty PickedPlanned
```

Comment:
why not subtract this anymore?

Reply:
I also feel a bit anxious about this.
do you know when the meaning of `qtyToDeliver` changed? Doesn't that impact other logic too?
Anyway, I suggest commenting it out (instead of removing) and adding a comment nearby (similarly to the other comments from here).

Reply:
done

Reply:
> do you know when the meaning of qtyToDeliver changed? Doesn't that impact other logic too?

Reply:
it's used only for picking job creation, and before this change, they were created with wrong data if there was something already picked for a shipment schedule